integration_tests/integration/make_a_referral/referralSectionVerifier.js

```diff
@@ -41,6 +41,14 @@ class _ReferralSectionChecker {
     return this
   }
 
+  selectServiceCagories(activeLinks) {
+    cy.get("[data-cy=url]:contains('Select service types')").should(
```

Comment:
i think 'types' isn't the right name here. i think 'service type' is what we are using to refer to the top level contract type, and these are 'service categories'.